src/main.py

```diff
@@ -585,53 +581,30 @@ def run(settings, components, variables):
   projectLength = getProjectLength(settings, components, v=v)
   vprint(v, 0, m, ' ... project length: {} years'.format(projectLength))
   projectCashflows = projectLifeCashflows(settings, components, lifetimeCashflows, projectLength, v=v)
-  compCashflows = {}
-  for comp in components:
-    tax = comp.getTax() if comp.getTax() is not None else settings.getTax()
-    inflation = comp.getInflation() if comp.getInflation() is not None else settings.getInflation()
-    compProjCashflows = projectComponentCashflows(comp, tax, inflation, lifetimeCashflows[comp.name], projectLength, v=v)
-    compCashflows[comp.name] = compProjCashflows
 
   vprint(v, 0, m, '='*90)
   vprint(v, 0, m, 'Economic Indicator Calculations')
   vprint(v, 0, m, '='*90)
   indicators = settings.getIndicators()
   outputType = settings.getOutput()
 
-
-  some_data = {**projectCashflows, **compCashflows }
+  results = {}
+  if 'NPV_search' in indicators:
+    metric = npvSearch(settings, components, projectCashflows, projectLength, v=v)
+    results['NPV_mult'] = metric
+  if 'NPV' in indicators:
+    metric = NPV(components, projectCashflows, projectLength, settings.getDiscountRate(), v=v)
+    results['NPV'] = metric
+  if 'IRR' in indicators:
+    metric = IRR(components, projectCashflows, projectLength, v=v)
+    results['IRR'] = metric
+  if 'PI' in indicators:
+    metric = PI(components, projectCashflows, projectLength, settings.getDiscountRate(), v=v)
+    results['PI'] = metric
+  results['outputType'] = outputType
 
   if outputType is True:
-    some_data = {**projectCashflows, **compCashflows }
```

Comment:
@worseliz I tinkered with this line a few ways, and it appears to not do anything, as `projectCashflows` and `compCashflows` are identical, so their keys overwrite each other and only contain one unique set of data anyway. I assume this was the result of testing and iteration that probably can be removed now; let me know if that's a bad assumption.